db/migrate/20170511191636_remove_container_from_model_name.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveContainerFromModelName < ActiveRecord::Migration[5.0]
+  def change
+    def change
```

Comment:
nested def by mistake?
I suspect that's why Travis errors act like the migration didn't happen.